packages/admin/README.md

```diff
@@ -8,7 +8,7 @@ Almost all commands require you to provide a `providerUrl` such as `https://rops
 The CLI connects to [AirnodeRrp.sol](https://github.com/api3dao/airnode/blob/master/packages/protocol/contracts/AirnodeRrp.sol) contract, which address is derived from the current chain.
 You can optionally specify the contract address yourself by providing optional `airnodeRrp` command argument with address of the deployed contract on your targeted chain.
 
-Commands that require `mnemonic` will make an on-chain transaction.
+Commands that require `mnemonic` will make an on-chain transaction (with the exception of `derive-sponsor-wallet`).
```

Comment:
I think there are more exceptions, no? All view calls don't require mnemonic, no?

Reply:
When I added that comment I saw only that one but let me double check. Actually after today's protocol-team meeting I need to refactor derive-sponsor-wallet again and it will no longer be an exception.

Reply:
deprecated